app/src/main/res/values/appboy.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="com_appboy_api_key">3a0a1188-ee1d-4027-92c5-98c6033ad5f0</string>
+    <string name="com_appboy_api_key"></string>
```

Comment:
Do we need to scrub (filter-branch) the apiKeys from the repo history as well?

Reply:
This is just an example integration and we don't want others to use our API keys.

Reply:
Ok, but the keys being visible in history is not a security issue?